nautobot_ssot/exceptions.py

```diff
@@ -0,0 +1,67 @@
+"""Custom Exceptions to be used with SSoT integrations."""
+
+
+class AdapterLoadException(Exception):
+    """Raised when there's an error while loading data."""
+
+
+class AuthFailure(Exception):
+    """Exception raised when authenticating to on-prem CVP fails."""
```

Comment:
Is this specific to CVP? If so, should it be called CVPAuthFailure?

Reply:
I could see it used for other integrations so if anything I'd update the docstring.